blog/config.toml

```
@@ -12,6 +12,7 @@ languages = [
     { code = "zh-TW" }, # Chinese (traditional)
     { code = "ja" }, # Japanese
     { code = "fa" }, # Persian
+	{ code = "ru" }, # Russian
```

Nit: change indentation to use spaces instead of tabs:

```suggestion
    { code = "ru" }, # Russian
```